src/transactor.js

```diff
@@ -150,11 +151,10 @@ export function transactor(firebase, handlers, todoTrxRef, closedTrxRef) {
             // here, so warn user about it)
             result = JSON.parse(JSON.stringify({result}))
           } catch (e) {
-            console.log(`Error for transaction id=${id}, result cannot be saved\n
-                        to firebase (result=${result})`)
+            console.log(`Error for transaction id=${id}, result cannot be saved\n to firebase (result=${result})`)
             result = {}
           }
-          set(closedTrxRef.child(trData.frbId), {data: trData, result})
+          set(closedTrxRef.child(trData.frbId), {data: trData, ...result})
```

Comment:
Why the splat? You may accidentally overwrite 'data' key.. is there any reason for this? However, I don't have a strong opinion on this one